configure.ac

```diff
@@ -333,9 +333,9 @@ AC_ARG_ENABLE([werror],
     [enable_werror=no])
 
 AC_ARG_ENABLE([external-signer],
-    [AS_HELP_STRING([--enable-external-signer],[compile external signer support (default is no, requires Boost::Process)])],
+    [AS_HELP_STRING([--enable-external-signer],[compile external signer support (default is yes, requires Boost::Process)])],
```

Comment:
When did we decide to turn this on by default? This is not a change that should be slipped in via the GUI repo.

Reply:
This commit has been part of this PR since before the GUI repo existed. I could split it out to the main repo though.

Reply:
In https://github.com/bitcoin/bitcoin/pull/16546 it defaulted to off, and I see no discussion of the change in this original PR (https://github.com/bitcoin/bitcoin/pull/16549).  It should probably be an explicit conversation in the main repo?